[css-grid] Updated meta-data tags to refer the relevant spec section.

javifernandez opened the pull request:
The alignment related tests are not being indexed correctly in the test suite. The reason is that none of these tests refer to the relevant spec section. 

This change adds the meta-data links to refer the appropriated spec section for each group of tests.

<!-- Reviewable:start -->

<!-- Reviewable:end -->

javifernandez committed:
Updated meta-data tags to refer the relevant spec section.

w3c-bots commented:
# Build PENDING

Started: None
Finished: None



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17495)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/265535354)

w3c-bots commented:
# Build ERRORED

Started: 2017-08-17 11:26:34
Finished: 2017-08-17 12:09:12



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17495)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/265535354)

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-010.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Content Distribution 'space-evenly' and gaps on 4x4 grid</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-012.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Content Distribution 'space-between' on 3x3 grid</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-017.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Content Distribution 'space-around' on 3x3 grid</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-024.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Content Distribution 'stretch' on 4x4 grid</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-005.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'space-evenly' and gaps on 2x2 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#collapsed-track">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-008.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'space-evenly' on 4x4 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#collapsed-track">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-009.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'space-evenly' and gaps on 4x4 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#collapsed-track">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-012.html:
```diff
@@ -1,9 +1,10 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Content Distribution 'space-between' and gaps on 3x3 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-012.html:
```diff
@@ -1,9 +1,10 @@
 <title>CSS Grid Layout Test: Content Distribution 'space-between' and gaps on 3x3 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#collapsed-track">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-013.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'space-between' on 4x4 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#collapsed-track">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-014.html:
```diff
@@ -2,8 +2,9 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Content Distribution 'space-between' and gaps on 4x4 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-015.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'space-around' and gaps on 2x2 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#collapsed-track">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-016.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'space-around' on 3x3 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#collapsed-track">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-018.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'space-around' on 4x4 grid with collapsed tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#collapsed-track">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-020.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'stretch' with collapsed tracks on 2x2 grid</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
-<link rel="help" href="https://drafts.csswg.org/css-align-3/#content-distribution">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-023.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'stretch' with collapsed tracks on 4x4 grid</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
-<link rel="help" href="https://drafts.csswg.org/css-align-3/#content-distribution">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-024.html:
```diff
@@ -2,8 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'stretch' and gaps with collapsed tracks on 4x4 grid</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
-<link rel="help" href="https://drafts.csswg.org/css-align-3/#content-distribution">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-002.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-003.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-004.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-005.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-006.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-007.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-008.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-009.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-011.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-012.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-013.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-014.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-015.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-016.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-001.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-002.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-003.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-004.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-005.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-006.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-007.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-008.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-009.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-010.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-011.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-012.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-013.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-014.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-015.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-lr-016.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-001.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-002.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-003.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-004.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-005.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-006.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-007.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-008.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on fixed-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-009.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-010.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-011.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-012.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-013.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-014.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-015.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-016.html:
```diff
@@ -2,9 +2,11 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Alignment and stretch on auto-sized tracks</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#alignment">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/grid-self-alignment-stretch-vertical-rl-016.html:
```diff
@@ -2,9 +2,11 @@
 <link rel="help" href="https://drafts.csswg.org/css-align-3/#propdef-align-self">
 <link rel="help" href="https://drafts.csswg.org/css-align-3/#propdef-justify-self">
 <link rel="help" href="https://drafts.csswg.org/css-align/#valdef-justify-self-stretch">
+<link rel="help" href="https://drafts.csswg.org/css-align/#valdef-align-self-stretch">
```
Same as above, use `css-align-3` rather than `css-align`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-001.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area height</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#column-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-002.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area height</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#column-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-003.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area height</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#column-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-004.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area height</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#column-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-005.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area height</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#column-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-006.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area height</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#column-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-007.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area width</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#row-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-008.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area width</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#row-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-009.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area width</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#row-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-010.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area width</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#row-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-011.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area width</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#row-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal commented on css/css-grid-1/alignment/self-baseline/grid-self-baseline-changes-grid-area-size-012.html:
```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8">
 <title>CSS Grid Layout Test: Self-Baseline alignment may change grid area width</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#row-align">
```
Same as above, use `css-grid-1` rather than `css-grid`.

frivoal reviewed (changes requested):
Nice patch. Just one little issue (repeated over all files), 1 typo, and 1 question, and we're good to go.

javifernandez commented on css/css-grid-1/alignment/grid-content-distribution-with-collapsed-tracks-001.html:
```diff
@@ -2,9 +2,9 @@
 <title>CSS Grid Layout Test: Content Distribution 'space-evenly' with collapsed tracks on 2x2 grid</title>
 <link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
-<link rel="help" href="https://drafts.csswg.org/css-align-3/#content-distribution">
-<link rel="help" href="https://drafts.csswg.org/css-align-3/#distribution-values">
```
Yes, as we don't include that info in the rest of the tests of this kind, which focus on the collapsed tracks issues. We already provide that info in the Content Distribution related tests. 

If you consider is still relevant info, I should add it to all these tests, for consistency.

frivoal replied:
I see. No, it's ok, no need to add it everywhere.

javifernandez commented:
Thanks for the review. I replied to your question and of course agree with the rest of the comments. I'll provide a new patch addressing all the issues ASAP.

javifernandez committed:
Use css-grid-1 rather than css-grid to refer the spec link

w3c-bots commented:
# Build PASSED

Started: 2017-08-17 14:54:53
Finished: 2017-08-17 15:31:22


<h2>Failing Jobs</h2>
<ul>

<li>build_css</li>

<li>chrome:unstable</li>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17510)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/265610372)

frivoal commented:
The patch fixes most of the problems, but not all. See the comments above that are still open.

javifernandez committed:
Additional link ref changes and fixed typo.

javifernandez commented:
Last patch addresses the pending issues.
